website/docs/kubernetes/lima/building-an-image-and-testing-it-in-lima.md

```diff
@@ -0,0 +1,58 @@
+---
+sidebar_position: 10
+title: Building and testing an image
+description: Building an image and testing it in Lima
+keywords: [podman desktop, podman, containers, pods, migrating, kubernetes, lima]
+tags: [migrating-to-kubernetes]
+---
+
+# Building an image and testing it in Lima
+
+With Podman Desktop, you can build an image with your container engine, and test it in your local Lima-powered Kubernetes cluster.
+
+#### Prerequisites
+
+- [You onboarded a container engine](/docs/onboarding-for-containers).
+- [You onboarded a Lima cluster](/docs/onboarding-for-kubernetes/lima).
+- [You have set your Kubernetes context to your local Lima-powered Kubernetes cluster](/docs/kubernetes/lima/working-with-your-local-lima-cluster).
+- A container definition file: `Containerfile` or `Dockerfile`.
+
+#### Procedure
+
+1. Build your image:
+
+   1. Open **<icon icon="fa-solid fa-cloud" size="lg" /> Images > <icon icon="fa-solid fa-cube" size="lg" /> Build an image**.
+   1. **Containerfile path**: select your `Containerfile` or `Dockerfile`.
+   1. **Build context directory**: optionally, select a directory different from the directory containing your `Containerfile` or `Dockerfile`.
+   1. **Image Name**: enter your image name `my-custom-image`.
+   1. Click **<icon icon="fa-solid fa-cube" size="lg" /> Build**.
+   1. Wait for the image build to finish.
+   1. Click **Done** to get back to the images list.
+
+1. Push your image to your Lima cluster:
+
+   1. **<icon icon="fa-solid fa-cloud" size="lg" /> Search images**: enter your image name `my-custom-image` to find the image.
+   1. Click **<icon icon="fa-solid fa-ellipsis-v" size="lg" /> > <icon icon="fa-solid fa-ellipsis-v" size="lg" /> Push image to Lima cluster**.
+
+1. Test your image by creating a container:
+
+   1. Click **<icon icon="fa-solid fa-play" size="lg" />** to open the **Create a container from image** dialog.
+   1. **Container name**: enter `my-custom-image-container`.
+   1. Review the parameters that Podman Desktop has detected from your image definition.
+   1. Click **<icon icon="fa-solid fa-play" size="lg" /> Start Container** to start the container in your container engine.
+
+1. Test your image and container on your Lima cluster:
+
+   1. **<icon icon="fa-solid fa-cloud" size="lg" /> Search containers**: enter `my-custom-image-container` to find the running container.
+   1. Click **<icon icon="fa-solid fa-ellipsis-v" size="lg" /> > <icon icon="fa-solid fa-rocket" size="lg" /> Deploy to Kubernetes** to open the **Deploy generated pod to Kubernetes** screen.
+   1. **Pod Name**: keep the proposed value `my-custom-image-container-pod`.
+   1. Optionally, if your container is exposing more than one port, select the port to expose.
+   1. **Kubernetes namespaces**: select `default`.
```

Comment:
```suggestion
   1. **Kubernetes Namespace**: select `default`.
```

Reply:
Copied from https://podman-desktop.io/docs/kind/building-an-image-and-testing-it-in-kind